### PR TITLE
FLUT-6329- [Feature] UG document for calendar

### PR DESCRIPTION
**Pull request by muthulakshmi-123**
## Description ##

I have added the user guide documentation for volume 1 features content of the flutter calendar. The prepared content was reviewed by @laksmipriyamurugesan 

The content review task link is followed by https://syncfusion.atlassian.net/browse/DOC-4172

### CODE REVIEW

File: Flutter/calendar/appointments.md [hunk 1]
```diff
@@ -408,6 +408,47 @@ class _AppointmentDataSource extends CalendarDataSource {
 
 ![Adding recurrence appointment](images/appointments/recurrence-appointment.png)
 
+### Monthly recurring appointments on last day of every month
+
+For creating monthly recurring appointments on last day of every month by using BYMONTHDAY=-1in the flutter event calendar.
```
Comment (Sasikumar3595) [inline Flutter/calendar/appointments.md:413]: Not monthly recurring appointment because it was used on yearly recurrence also

Comment (muthulakshmi-123) [reply Flutter/calendar/appointments.md]: I have checked and modified the content. I have updated in my latest commit

File: Flutter/calendar/timeslot-views.md [hunk 2]
```diff
@@ -530,6 +530,31 @@ Widget build(BuildContext context) {
 
 ![Allday panel Background color](images/getting-started/alldayPanelColor.jpg)
 
+## Number of days in view
+You can customize the days count, by setting the [numberOfDaysInView]() property of `timeSlotViewSettings` in the calendar.
```
Comment (laksmipriyamurugesan) [inline Flutter/calendar/timeslot-views.md:534]: You can customize the days count by setting the [numberOfDaysInView]() property of `timeSlotViewSettings` in the calendar.

File: Flutter/calendar/timeslot-views.md [hunk 3]
```diff
@@ -530,6 +530,31 @@ Widget build(BuildContext context) {
...
+    );
+  }
+
+{% endhighlight %}
+{% endtabs %}
+
+![Number of days in view](images/timeslot-views/numberOfDaysInView.jpg)
+
+>**NOTE**
+* It is applicable for day, week, workweek, timeline day, timeline week and timeline workweek views.
```
Comment (laksmipriyamurugesan) [inline Flutter/calendar/timeslot-views.md:556]: It is applicable for day, week, workweek, timeline day, timeline week, and timeline workweek views.

File: Flutter/calendar/appointments.md [hunk 4]
```diff
@@ -408,6 +408,47 @@ class _AppointmentDataSource extends CalendarDataSource {
 
 ![Adding recurrence appointment](images/appointments/recurrence-appointment.png)
 
+### Recurrence appointments on last day of month
+
+For creating recurring appointments on last day of month by using BYMONTHDAY=-1 in the flutter event calendar.
```
[the remark this answers is not in the document]
Comment (muthulakshmi-123) [reply Flutter/calendar/appointments.md]: I have checked and updated in my latest commit

File: Flutter/calendar/timeslot-views.md (hunk 2 above)
Comment (muthulakshmi-123) [reply Flutter/calendar/timeslot-views.md]: I have checked and updated in my latest commit

File: Flutter/calendar/timeslot-views.md (hunk 3 above)
Comment (muthulakshmi-123) [reply Flutter/calendar/timeslot-views.md]: I have updated this in my latest commit

File: Flutter/calendar/appointments.md [hunk 5]
```diff
@@ -408,6 +408,47 @@ class _AppointmentDataSource extends CalendarDataSource {
 
 ![Adding recurrence appointment](images/appointments/recurrence-appointment.png)
 
+### Recurrence appointments on last day of month
```
Comment (Nijamudeen-Mohamed) [inline Flutter/calendar/appointments.md:411]: Add this topic under **Flexible working days and working hours**

**Review by Nijamudeen-Mohamed — APPROVED** [review]

File: Flutter/calendar/appointments.md (hunk 5 above)
Comment (muthulakshmi-123) [reply Flutter/calendar/appointments.md:411]: I have checked and I have updated the topic (number of days in view) is under flexible working days and working hours. I have updated my changes in latest commit.

**Review by Nijamudeen-Mohamed — APPROVED** [review]

**Pull request merged**